crates/wasi-crypto/build.rs

```diff
@@ -0,0 +1,15 @@
+//! This build script:
+//!  - has the configuration necessary for the wiggle and witx macros.
+
+use std::path::PathBuf;
+
+fn main() {
+    // This is necessary for Wiggle/Witx macros.
+    let wasi_root = PathBuf::from("./spec").canonicalize().unwrap();
```

Comment:
Instead of using `canonicalize` you can use `env::current_dir().join("spec")` to geet an absolute path heere I believe.

Reply:
Changed to `env::current_dir()`. Thanks!